pkgs/applications/accessibility/dasher/default.nix

```diff
@@ -0,0 +1,55 @@
+{ stdenv, lib, fetchFromGitHub
+, autoreconfHook, pkgconfig, wrapGAppsHook
+, gnome3, glib, intltool, gtk3, gnome-doc-utils
+, at-spi2-core, dbus
+, libxslt, libxml2
+, speechSupport ? true, speechd ? null
+}:
+
+assert speechSupport -> speechd != null;
+
+stdenv.mkDerivation {
+  pname = "dasher";
+  version = "2018-04-03";
+
+  src = fetchFromGitHub {
+    owner = "dasher-project";
+    repo = "dasher";
+    rev = "9ab12462e51d17a38c0ddc7f7ffe1cb5fe83b627";
+    sha256 = "1r9xn966nx3pv2bidd6i3pxmprvlw6insnsb38zabmac609h9d9s";
+  };
+
+  prePatch = ''
+    # tries to invoke git for something, probably fetching the ref
+    echo "true" > build-aux/mkversion
+  '';
+
+  configureFlags = lib.optional (!speechSupport) "--disable-speech";
+
+  nativeBuildInputs = [
+    autoreconfHook
+    wrapGAppsHook
+    pkgconfig
+    # doc generation
+    gnome-doc-utils
+    libxslt libxml2
+  ];
+
+  buildInputs = [
+    gnome3.gnome-common
```

Comment:
I didn't need gnome-common to build this, and given that's completely deprecated we shouldn't use it.
(plus it's not a library)

Reply:
however it does propagate `which` so you do need to add that to build this.

Reply:
I saw it in some of the output, and it’s used by lots of other packages. Gnome stuff is a walk in the dark for me.